updateStableIds/runUpdateStableIds.sh

```diff
@@ -0,0 +1,9 @@
+## Update repo
```

Comment:
Could this be added to the beginning of the bash script to allow it to be called from any directory:

```
#!/bin/bash
DIR=$(dirname "$(readlink -f "$0")") # Directory of the script -- allows the script to invoked from anywhere
cd $DIR
```

Reply:
Is it expected that this script will be invoked from _any_ other directories?

Reply:
I'm not sure yet, but I think it should have the ability to be invoked from elsewhere since it is the entrypoint to the entire program for the end-user.  We may need to invoke from another working directory as part of the pipeline and it's only a small addition to the script.

Reply:
Done